relay-event-normalization/src/normalize/mod.rs

```diff
@@ -990,6 +1060,49 @@ impl<'a> DynamicMeasurementsConfig<'a> {
     }
 }
 
+/// Defines a weighted component for a performance score
+///
+/// Weight is the % of score it can take up (eg. LCP is a max of 35% weight for desktops)
+/// Currently also contains (p10, p50) which are used for log CDF normalization of the weight score
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+pub struct PerformanceScoreWeightedComponent {
```

Comment:
be careful about using 'pub' too much, it seems here most of your data can be internal but it's marked as pub